[2025-03-19T07:06:39.885+0000] {local_task_job_runner.py:115} ERROR - Received SIGTERM. Terminating subprocesses
[2025-03-19T07:06:39.886+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 143
[2025-03-19T07:06:39.901+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-03-19T07:06:39.913+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_pipeline.extract scheduled__2025-03-19T06:00:00+00:00 [queued]>
[2025-03-19T07:06:39.919+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_pipeline.extract scheduled__2025-03-19T06:00:00+00:00 [queued]>
[2025-03-19T07:06:39.920+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2025-03-19T07:06:39.929+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): extract> on 2025-03-19 06:00:00+00:00
[2025-03-19T07:06:39.932+0000] {standard_task_runner.py:57} INFO - Started process 184 to run task
[2025-03-19T07:06:39.934+0000] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'etl_pipeline', 'extract', 'scheduled__2025-03-19T06:00:00+00:00', '--job-id', '52', '--raw', '--subdir', 'DAGS_FOLDER/etl_dags.py', '--cfg-path', '/tmp/tmp2553m9ph']
[2025-03-19T07:06:39.935+0000] {process_utils.py:131} INFO - Sending 15 to group 184. PIDs of all processes in the group: [184]
[2025-03-19T07:06:39.936+0000] {standard_task_runner.py:85} INFO - Job 52: Subtask extract
[2025-03-19T07:06:39.936+0000] {process_utils.py:86} INFO - Sending the signal 15 to group 184
[2025-03-19T07:06:39.938+0000] {process_utils.py:79} INFO - Process psutil.Process(pid=184, status='terminated', exitcode=<Negsignal.SIGTERM: -15>, started='07:06:39') (184) terminated with exit code -15
[2025-03-19T07:06:43.219+0000] {taskinstance.py:1149} INFO - Dependencies not met for <TaskInstance: etl_pipeline.extract scheduled__2025-03-19T06:00:00+00:00 [running]>, dependency 'Task Instance Not Running' FAILED: Task is in the running state
[2025-03-19T07:06:43.224+0000] {taskinstance.py:1149} INFO - Dependencies not met for <TaskInstance: etl_pipeline.extract scheduled__2025-03-19T06:00:00+00:00 [running]>, dependency 'Task Instance State' FAILED: Task is in the 'running' state.
[2025-03-19T07:06:43.225+0000] {local_task_job_runner.py:154} INFO - Task is not able to be run
